install.sh

```diff
@@ -26,7 +26,16 @@ else
 		# * peg/leg is compiled by CMake from sources and used during build without installing
 		#   that is because I did not find it packaged.
 		echo "    + Installing yum packages"
-		$SUDO yum install -y gc-devel libffi-devel json-c-devel pcre-devel make cmake3 ctest pandoc pkgconfig
+		$SUDO yum install -y gc-devel libffi-devel json-c-devel pcre-devel make cmake3 pandoc pkgconfig
+		if ! command -v ctest &>/dev/null;then
+		  if command -v ctest3 &>/dev/null;then
```

Comment:
Please merge into one `if` if possible

Reply:
not possible, because the else case needs to happen only if inside the first if